### DISCUSSION TITLE
Use Freeleech Tokens

### DISCUSSION THREAD

**Discussion by jay-kub** [autobrr/autobrr#573] [Ideas]
It would be great if Autobrr had an option to use freeleech tokens when downloading torrent files.

If anyone knows a workaround to get this setup please share. If I could trigger Autobrr to use Jackett to download the torrent and put it in my qbittorrent watch directory that would also work, but it doesn't appear jackett is supported yet. Unfortunately using Lidarr is not an option as I use autobrr for concert recordings and they turn into an import disaster on Lidarr.

Edit: jackett/Prowlarr are supported as indexers but not as download clients, meaning they couldn't be used as a workaround to download the torrent when triggered by an IRC indexer. Since jackett/Prowlarr are RSS feeds, they are too slow to grab torrents when racing.

**Comment by KyleSanderson** [comment]
Jackett is fully supported - can you ping me on Discord?

**Comment by pepijnfrenken** [comment]
Have there been any updates on this feature?

**Comment by Julianwan** [comment]
Is this possible through Prowlarr?

**Comment by jonandeva** [comment]
I would love this feature on RED, if possible.

**Comment by s0up4200** [comment]
Automating the use of freeleech tokens is against the rules on RED, and probably on OPS too. I don't like the idea of going against that, personally.

**Golden Rule 5.3 on RED:**
> Do not autosnatch freeleech torrents.
> The automatic snatching of freeleech torrents using any method involving little or no user-input (e.g., API-based scripts, log or site scraping, etc.) is prohibited.

**Comment by jonandeva** [comment]
Agreed. I thought I saw something about that. Too bad I guess. Thanks for looking into it.

**Reply by jay-kub** [reply]
@s0up4200 @jonandeva 

I believe there is a misunderstanding. RED prohibits the autosnatching of freeleech torrents, however this is different from autosnatching torrents using freeleech tokens. Autobrr would be snatching regular torrents, and using freeleech tokens to do so, if available. This would not be against RED's rules, or any other tracker afaik.

**Comment by nebb00** [comment]
It's not against the rules. But would also love this feature, esp for RED

**Comment by s0up4200** [comment]
We reached out to the staff at RED yesterday to get a statement about this. They pretty much confirmed our suspicion.

While the possibility to do it exists, its not something we will encourage users to do. Always make sure you respect the rules of any tracker that you are a part of.

Will obviously leave this thread open as we might implement features to make it work for trackers that *do* allow it.

**Reply by jay-kub** [reply]
Gotcha, my apologies then. Appreciate you looking into it and everything you do @s0up4200

**Reply by anonbaby** [reply]
> We reached out to the staff at RED yesterday to get a statement about this. They pretty much confirmed our suspicion.
> 
> While the possibility to do it exists, its not something we will encourage users to do. Always make sure you respect the rules of any tracker that you are a part of.
> 
> Will obviously leave this thread open as we might implement features to make it work for trackers that _do_ allow it.

OPS allows it. 
see ops/forums.php?action=viewthread&threadid=13703&postid=231496#post231496

**Reply by jgosnell56** [reply]
Looks like AR allows it as well.  This would be a great feature to have.
